### ISSUE TITLE
Does EarlyStopping with a fixed base exist?

### ISSUE THREAD

**Issue by RnoldR** [keras-team/keras#11496]
Please make sure that the boxes below are checked before you submit your issue.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`

- [ ] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).
Got 1.9.0 as 1.10.0 causes problems

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

I want to  stop the training when accuracy 'acc' reaches a certain hard limit, let's say: 0.9. In the manual it is suggested that that creating an instance of EarlyStopper with the baseline parameter should do it, e.g. like this:
        stopper = EarlyStopping(
                monitor = 'acc',
                verbose = 0.1,
                mode = 'max',
                baseline = 0.9
                )

In my case the training stops immediately. Looking at the code in callback.py it could never work (see line 538-540 in keras 2.2.2). Is my understanding of the baseline parameter incorrect or is it the code?
